reference/composedshader.md

```diff
@@ -17,10 +18,21 @@ If several [ShaderPart](shaderpart.md) nodes of the same type are given, then on
 
 The `uniforms` field is a list of [Uniform](uniform.md) nodes corresponding to the uniform variables passed to the [ShaderPart](shaderpart.md) programs.
 
+The `transparent` field indicates to Webots if its fragment program plans to use the alpha channel.
```

Comment:
This sentence is not very clear to me... I believe that the "to Webots" is superfluous. What is the "fragment program"? Has it a plan?...